python/cuml/test/test_preprocessing.py

```diff
@@ -421,10 +439,18 @@ def test_add_dummy_feature(clf_dataset, value):  # noqa: F811
 
 
 @pytest.mark.parametrize("value", [1.0, 42])
+@pytest.mark.parametrize("sp_format", ["coo", "csr", "csc"])
 def test_add_dummy_feature_sparse(sparse_dataset_with_coo,  # noqa: F811
-                                  value):
+                                  value, sp_format):
     X_np, X = sparse_dataset_with_coo
 
+    if sp_format == "csc":
+        X_np = X_np.tocsc()
+        X = X.tocsc()
+    elif sp_format == "csr":
+        X_np = X_np.tocsr()
+        X = X.tocsr()
```

Comment:
This change shouldn't be necessary as `sparse_dataset_with_coo` already implements the CSR, CSC and COO formats in Scipy and Cupy arrays.